### PR TITLE
[Ready] Fixed script objects falling through the ground

### PR DESCRIPTION
**Pull request by husho**
This fixes issue when during the game intro few objects fall through the ground.

- [x] 0107 opcode
- [x] 0392 opcode​

Issues to fix:
- [x] Lamposts act weirdly now, seems like they only fall down after 2nd hit
- [x] Hydrants don't longer break

### CODE REVIEW

**Comment by mythrillex** [comment]
After the fix, some hydrants dont break when u hit, its normal ?

**Comment by husho** [comment]
I'll test

**Comment by husho** [comment]
What you described happens in master branch, in my branch hydrants don't even break

File: rwengine/src/render/ObjectRenderer.cpp [hunk 1]
```diff
@@ -308,7 +308,8 @@ void ObjectRenderer::renderVehicle(VehicleObject* vehicle,
 void ObjectRenderer::renderPickup(PickupObject* pickup, RenderList& outList) {
     if (!pickup->isEnabled()) return;
 
-    glm::mat4 modelMatrix = glm::translate(glm::mat4(1.0f), pickup->getPosition());
+    glm::mat4 modelMatrix =
+        glm::translate(glm::mat4(1.0f), pickup->getPosition());
```
Comment (danhedron) [inline rwengine/src/render/ObjectRenderer.cpp]: Looks like there's just formatting changes in this file

File: rwengine/src/script/modules/GTA3ModuleImpl.inl [hunk 2]
```diff
@@ -10395,10 +10396,8 @@ void opcode_0381(const ScriptArguments& args, const ScriptObject object, ScriptV
     @arg arg2 Boolean true/false
 */
 void opcode_0382(const ScriptArguments& args, const ScriptObject object, const ScriptBoolean arg2) {
-    RW_UNIMPLEMENTED_OPCODE(0x0382);
-    RW_UNUSED(object);
-    RW_UNUSED(arg2);
     RW_UNUSED(args);
+	object->setStatic(!arg2);
```
Comment (danhedron) [inline rwengine/src/script/modules/GTA3ModuleImpl.inl]: The description of this opcode seems to be that it turns collision detection on and off, not change static status.

File: rwengine/src/render/ObjectRenderer.cpp (hunk 1 above)
Comment (husho) [reply rwengine/src/render/ObjectRenderer.cpp]: getVisible was placed with isVisible

File: rwengine/src/script/modules/GTA3ModuleImpl.inl (hunk 2 above)
Comment (husho) [reply rwengine/src/script/modules/GTA3ModuleImpl.inl]: You right. Will fix.

**Pull request merged**